src/Runner.Sdk/RunnerWebProxy.cs

```diff
@@ -165,6 +165,12 @@ public RunnerWebProxy()
                             continue;
                         }
 
+                        // Strip leading '.' fron noproxy host
+                        if (noProxyInfo.Host.StartsWith("."))
+                        {
+                            noProxyInfo.Host = noProxyInfo.Host[1..];
```

Comment:
What by mistake someone set the host to sth like "cs."?

Reply:
Like `no_proxy="example.com, cs."`? 
I don't think any valid Hostnames end with `cs.`, end with a `.`. So, `cs.` would match nothing.

Reply:
Aha this is a range operation, so if the host will be equal to `cs.` this logic will set noProxyInfo.Host to just `.` 
It's definitely not a valid host but it could be added by mistake. :)